accProcess.py

```diff
@@ -1,5 +1,6 @@
 """Command line tool to extract meaningful health info from accelerometer data."""
 
+import dxpy
```

Comment:
Is this part of the PR?

Reply:
Thanks @chanshing, I can confirm this is not part of the PR. My last two minor commits were used to rectify this. Let me see if there is a way that I can squash these into the first commit...